pytorch_lightning/accelerators/accelerator_connector.py

```diff
@@ -542,6 +546,27 @@ def _set_horovod_backend(self):
         else:
             self.num_processes = hvd.local_size()
 
+    def check_ipython_compatibility(self):
+        """Raises a `MisconfigurationException` if the accelerator is not compatible with IPython and code is run in an IPython kernel."""
+        if self._distrib_type not in self.ipython_compatible_distrib_types:
```

Comment:
you can invert the if statement and return early.

Reply:
Done